src/main/java/Jarvis.java

```diff
@@ -0,0 +1,143 @@
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+import java.io.FileWriter;
+
+public class Jarvis {
```

Comment:
In general across all the files, adding Javadoc comments would make the content clearer.